src/components/Login/Login.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import {
+  View,
+  Text,
+} from 'react-native';
+import TextStyles from '../../helpers/TextStyles';
+import TextField from '../common/TextField';
+import ShadowStyles from '../../helpers/ShadowStyles';
+import Button from '../common/Button';
+import styles from './styles';
+import App from '../../App';
```

Comment:
Just a detail, but what do you think about sorting the imports by the path? For example, first, the ones with `../../helpers`, then `../../`, then `../common` and then the ones on the same path `./`

(like you did on Profile.js)

Reply:
Sounds like a good idea, it would be nice if there was some kind of eslint rule for that (so we don't need to check by hand 😅 )

Reply:
Maybe there is, I will investigate.